### PR TITLE
More array fun

### PR DESCRIPTION
**Pull request by epuzio**
tests are now formatted as: “reversePrinterWorks_fourStrings” for a
single case as opposed to “testEvenPrinter” covering all test cases.

### CODE REVIEW

File: src/_03_More_Array_Fun/MoreArrayFun.java [hunk 1]
```diff
@@ -6,44 +6,74 @@
...
+	 * 
+	 * /** DO THIS NEXT WEEK 5. Write a method that takes an array of Strings and
+	 * prints all the Strings in the array in a completely random order. Almost
+	 * every run of the program should result in a different order.
+	 * 
+	 * @return
+	 * 
+	 * 
+	 * 		}
+	 *
```
Comment (IsisJ) [inline src/_03_More_Array_Fun/MoreArrayFun.java]: Do you still need all of the comments❓

File: src/_03_More_Array_Fun/MoreArrayFun.java [hunk 2]
```diff
@@ -6,44 +6,74 @@
...
+		String allWords = "";
+		for (int i = 0; i < array.length; i++) {
+			if (i == array.length - 1) {
+				allWords = allWords + array[i];
+			} else {
+				allWords = allWords + array[i] + ", ";
+			}
+		}
+		return allWords;
 	}
```
Comment (IsisJ) [inline src/_03_More_Array_Fun/MoreArrayFun.java:19]: Clear and concise! Looking good 👍

File: src/_03_More_Array_Fun/MoreArrayFunTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	}
+
+	@Test
+	public void reversePrinterWorks_blankString() {
+		String[] arrayFour = { "" };
+		String allValuesFour = MoreArrayFun.stringArrayReverseOrderPrinter(arrayFour);
+		assertEquals("", allValuesFour);
+	}
+
+}
```
Comment (IsisJ) [inline src/_03_More_Array_Fun/MoreArrayFunTest.java:110]: Do all of your tests work in every scenario?

**Review by IsisJ — COMMENTED** [review]
Maybe you should check your tests and get rid of any unneeded comments

File: src/_03_More_Array_Fun/MoreArrayFunTest.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+		String[] array = { "blue", "cello", "cactus", "electric" };
+		String allValues = MoreArrayFun.stringArrayPrinter(array);
+		assertEquals("blue, cello, cactus, electric", allValues);
+
+	}
+
+	@Test
+	public void arrayPrinterWorks_twoStringsWithSpaces() {
+
+		String[] arrayTwo = { "snow leopard enclosure", "bean paste candy" };
```
Comment (mwoguri) [inline src/_03_More_Array_Fun/MoreArrayFunTest.java]: This probably doesn't need to be called `arrayTwo`.  `array` would work fine.

File: src/_03_More_Array_Fun/MoreArrayFun.java (hunk 1 above)
Comment (epuzio) [reply src/_03_More_Array_Fun/MoreArrayFun.java]: nope!

File: src/_03_More_Array_Fun/MoreArrayFunTest.java (hunk 4 above)
Comment (epuzio) [reply src/_03_More_Array_Fun/MoreArrayFunTest.java]: 👍

**Review by mwoguri — APPROVED** [review]
LGTM!  Please merge.

**Pull request merged**